client/state/ui/first-view/selectors.js

```diff
@@ -57,7 +57,7 @@ export function isViewEnabled( state, config ) {
 		return false;
 	}
 
-	const firstViewHistory = getPreference( state, 'firstViewHistory' ).filter( entry => entry.view === config.name );
+	const firstViewHistory = filter( getPreference( state, 'firstViewHistory' ), entry => entry.view === config.name );
```

Comment:
Not blocker, but this could be expressed with Lodash's object matching:

```js
filter( getPreference( state, 'firstViewHistory' ), { view: config.name } );
```